src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition/help-lightweight.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    If selected, try to obtain the Pipeline script contents directly from the SCM without performing a full checkout.
+    The advantage of this mode is its efficiency; however, you will not get any changelogs or polling based on the SCM.
```

Comment:
Would require a significant rewrite. Currently `workflow-job` knows to record changelogs, or consider polling, based on `SCMListener` methods which cannot be called in lightweight mode. I considered such a large change (APIs etc.) out of scope here, especially given that

* many people will not care about changelogs or polling for the SCM containing the script
* if you `checkout scm` the same repo later, you will get those anyway